business/workloads.go

```diff
@@ -135,7 +148,8 @@ func (in *WorkloadService) GetWorkloadList(criteria WorkloadCriteria) (models.Wo
 		err = <-errChan
 		return *workloadList, err
 	}
-
+	authpolicies = istioConfigList.AuthorizationPolicies
+	validations := in.getWorkloadValidations(authpolicies, *workloadList, criteria.Namespace)
```

Comment:
It looks like there was one message that didn't arrive into my last review. Apologies.
There is a problem here. workloadList is still an empty struct at this point. It is actually filled in in the line 160, right at the end of the loop (`workloadList.Workloads = append(workloadList.Workloads, *wItem)`)
If you place this selected line after the loop, where you have the Validation assignation, it should work.

By checking the following endpoints you should see the results:
`/api/namespaces/bookinfo/workloads?validate=true` for Workload List
`/api/namespaces/bookinfo/workloads/details-v1?validate=true` for the details page.

I don't know if you have the local environment fully installed, here you have some more details: https://github.com/kiali/kiali#developer-setup

And if you need any further assistance, I can help. You can find me on the istio slack if you need me.